v0100/smlrc.c

```diff
@@ -1153,7 +1154,7 @@ void IncludeFile(int quot)
       }
       for (i = 0; i < pl; )
       {
-        int plen = strlen(paths + i);
+        size_t plen = strlen(paths + i);
```

Comment:
Which version of gcc, what options (just doing make?), what's the warning message?